docker-compose.yml

```diff
@@ -364,6 +364,25 @@ services:
     restart: always
     container_name: os_dev_phpmyadmin
 
+  openldap:
+    image: osixia/openldap:1.5.0
```

Comment:
FYI: Seems not maintained last year.
https://hub.docker.com/r/bitnami/openldap seems maintaining